indicators/credicorp-bank-tgeblg.yml

```diff
@@ -0,0 +1,38 @@
+title: Credicorp Bank Phishing Kit tGeBlg
+description: |
+    Detects a phishing kit targeting Credicorp Bank.
+    This was found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://www.credicorpbank.com/
+    - https://urlscan.io/result/c3582f08-595f-4796-8cb6-a794fdf13c2d/
+    - https://urlscan.io/result/5fc51b7e-6122-427b-9d06-c8e6f16f4b8f/
+    - https://urlscan.io/result/2684e40f-2619-46b9-9206-1fdb2e57b38a/
+
+detection:
+
+    headerCapitalization:
+      html|contains|all:
+        - meta http-equiv="CACHE-CONTROL" content="NO-CACHE"
+        - meta http-equiv="PRAGMA" content="NO-CACHE"
+
+    dojoJS:
```

Comment:
What may be better instead of this, is detecing the `SESSION_CSRF_TOKEN` which has the value `0f234c429f530238b63c6793ca994d0c85c6f37b`